.github/workflows/ci.yaml

```diff
@@ -39,7 +39,13 @@ jobs:
         with:
           provider: lxd
       - name: Run integration tests
-        run: tox -e charm-integration
+        run: |
+          # set sysctl values in case the cloudinit-userdata not applied
```

Comment:
is this really needed?

Reply:
For this specific test it's not (it's required starting from where opensearch gets started), but I wanted as much as possible the test environment to stay consistent across the multiple tests